src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
I don't understand... why filter that out? the user has to select that backgroung_page in order to debug it using chrome dev tools... It is also a diff URL, so the user can explicitly target that URL if they want on launch.json... Also, that doesn't mess with the officially supported scenario since Pages are still supported and not affected by that. The only users who will ever have to use that, are the ones developing NW.js or Electron applications. I don't see how it can bring problem but its ok, take your time. I'm working with the hacked script while it isn't published to the official extension, so no rush :)